00-main.go

```diff
@@ -21,16 +22,24 @@ func main() {
 		"prefix for request URLs",
 	)
 
+	flag.BoolVar(
+		&skipTLSVerification,
+		"skipTLSVerification",
```

Comment:
command line options are always lowercase and hyphens, so let's make this: skip-tls-verification